### DISCUSSION TITLE
Shifting stroke pattern

### DISCUSSION THREAD

**Discussion by arkadiuszwojcik** [cetz-package/cetz#565] [Q&A] [answered]
Is it possible to set stroke pattern shift? I have dashed stroke pattern like on image below, can I shift top horizontal line pattern left a bit so I can have nice corners?

![Screenshot 2024-04-23 100424](https://github.com/cetz-package/cetz/assets/2368976/8fa72737-84ab-4078-add3-319908986b26)

**Comment by fenjalien** [comment, marked as answer]
Not directly through cetz, but it uses Typst's stroke: https://typst.app/docs/reference/visualize/stroke/ so you could try defining your own dash pattern using arrays or dictionarys
